ManatiCR/vitaminau
# VIT-75: Conditional fields registry.

# Task [VIT-75](https://manati.atlassian.net/browse/VIT-75).
Show specific unit field when selecting "No encuentro mi unidad" in the unit catalog.

#Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the following url: 
  - http://vitaminau.docker/user/register
- Go to the section "Sobre su trabajo", in the field "Unidad" select "No encontré mi unidad".
  - The "Unidad especifica" field should be displayed.
- Press the "Añadir información de unidad" button and repeat the previous steps but in the "Unidad" field, select any option except "No encontré mi unidad"
  - The "Unidad especifica" field should not be displayed.
- Fill in all the fields to register and press the button "Crear nueva cuenta"
  - Should register correctly

# CA
- The "Unidad especifica" field should be displayed only when "No encontré mi unidad" is selected in the "Unidad" field.
- User can be registered successfully.
